.gitignore

```diff
@@ -0,0 +1,2 @@
+/go-wait-for-it
+/go.sum
```

Comment:
Usually this gets checked in, as it allows for reproducible builds.

Reply:
I noticed a pile of lines for non-dependency packages in that file; the file is 285 lines in total. While I may not have anything secret in there, it felt like a violation of privacy. Any ideas about how to trim that file down to the list of all direct and indirect dependencies, only?

Reply:
It should be direct and indirect dependencies. It only contains those, nothing more. I don't see how that should violate your privacy honestly (I'm curious). Basically all Go projects have these.

Reply:
Okay, let's see If I can prove or disprove that these 285 contain unrelated packages. I have an idea, give me a few minutes.

Reply:
I tried it in fresh Debian container now and I get the same 285 lines for `go.sum` in there. Alright, I'll add the file.